osu.Game/Rulesets/Scoring/ScoreProcessor.cs

```diff
@@ -18,6 +18,7 @@ public class ScoreProcessor : JudgementProcessor
     {
         private const double base_portion = 0.3;
```

Comment:
think we should rename this to `accuracy_portion`? not sure if it works better than "base"..

Reply:
I was going to do that in the upcoming mania PR but can do it here if you'd rather that.

Reply:
Either or.